packages/zoe/src/contracts/priceAggregator.js

```diff
@@ -590,8 +590,10 @@ const start = async (zcf, privateArgs) => {
 
       // Obtain the oracle's publicFacet.
       assert(oracleInstance);
-      /** @type {import('./oracle.js').OracleContract['publicFacet']} */
-      const oracle = await E(zoe).getPublicFacet(oracleInstance);
+      const oracle =
+        /** @type {import('./oracle.js').OracleContract['publicFacet']} */ (
+          await E(zoe).getPublicFacet(oracleInstance)
+        );
```

Comment:
the return from `getPublicFacet` is no longer assignable to the Oracle's publicFacet?

Reply:
@Chris-Hibbert I'm interested in your take on this.

Reply:
I'd like to see this change noted explicitly in the PR description.